doc/instances.md

```diff
@@ -755,60 +755,60 @@ lxc config device add <instance> <device-name> infiniband nictype=sriov parent=<
-required            | boolean   | true      | no        | Controls whether to fail if the source doesn't exist
-readonly            | boolean   | false     | no        | Controls whether to make the mount read-only
-size                | string    | -         | no        | Disk size in bytes (various suffixes supported, see {ref}`instances-limit-units`). This is only supported for the rootfs (/)
-size.state          | string    | -         | no        | Same as size above but applies to the filesystem volume used for saving runtime state in virtual machines.
-recursive           | boolean   | false     | no        | Whether or not to recursively mount the source path
-pool                | string    | -         | no        | The storage pool the disk device belongs to. This is only applicable for storage volumes managed by LXD
-propagation         | string    | -         | no        | Controls how a bind-mount is shared between the instance and the host. (Can be one of `private`, the default, or `shared`, `slave`, `unbindable`,  `rshared`, `rslave`, `runbindable`,  `rprivate`. Please see the Linux Kernel [shared subtree](https://www.kernel.org/doc/Documentation/filesystems/sharedsubtree.txt) documentation for a full explanation) <!-- wokeignore:rule=slave -->
-shift               | boolean   | false     | no        | Set up a shifting overlay to translate the source uid/gid to match the instance (only for containers)
-raw.mount.options   | string    | -         | no        | Filesystem specific mount options
-ceph.user\_name     | string    | admin     | no        | If source is Ceph or CephFS then Ceph user\_name must be specified by user for proper mount
-ceph.cluster\_name  | string    | ceph      | no        | If source is Ceph or CephFS then Ceph cluster\_name must be specified by user for proper mount
-boot.priority       | integer   | -         | no        | Boot priority for VMs (higher boots first)
-
-#### Type: unix-char
+`limits.read`       | string    | -         | no        | I/O limit in byte/s (various suffixes supported, see {ref}`instances-limit-units`) or in IOPS (must be suffixed with `iops`) - see also {ref}`storage-configure-IO`
+`limits.write`      | string    | -         | no        | I/O limit in byte/s (various suffixes supported, see {ref}`instances-limit-units`) or in IOPS (must be suffixed with `iops`) - see also {ref}`storage-configure-IO`
+`limits.max`        | string    | -         | no        | Same as modifying both limits.read and limits.write
+`path`              | string    | -         | yes       | Path inside the instance where the disk will be mounted (only for containers).
+`source`            | string    | -         | yes       | Path on the host, either to a file/directory or to a block device
+`required`          | Boolean   | true      | no        | Controls whether to fail if the source doesn't exist
```

Comment:
The capitlisation of Boolean looks odd. Perhaps we can add an exception to allow "boolean", or make the others capitlised?

Reply:
Boolean is usually capitalized (since it stems from a name), so I don't see this as a problem.
We should not capitalize the other types though, that is just wrong.

We could change all occurrences in the table to "bool", but I'm not sure it's worth it?

Reply:
Let's go with "bool" then.
All official dictionaries have Boolean capitalised, so I really don't want to add a wrong spelling as an exception. And overall, I'm always against capitalizing anything that doesn't need to be capitalized (thus other data type names). So "bool" is the best compromise I guess. :)

Reply:
Sounds good thanks. I was looking at other technical documentation for projects that have data types and they seem to either capitalise first letter of all data types or use "bool", so seems like a common approach.

https://tip.golang.org/ref/spec#Boolean_types


Its funny that "Boolean" is consider to be capitalised, because it relates to George Boole but "bool" doesn't, even though its closer to his name. :)